src/pages/iou/MoneyRequestAmountForm.tsx

```diff
@@ -261,7 +262,7 @@ function MoneyRequestAmountForm(
                     amount={amount}
                     currency={currency}
                     isCurrencyPressable={isCurrencyPressable}
-                    onCurrencyButtonPress={onCurrencyButtonPress}
+                    onCurrencyButtonPress={() => onCurrencyButtonPress?.(moneyRequestAmountInput.current?.getCurrency())}
```

Comment:
This is not what I meant. We shouldn't pass any prop to `onCurrencyButtonPress`, instead on `IOURequestStepAmount` we should use the input ref's method `getCurrency`

Reply:
If we do that we also need to add a prop `moneyRequestAmountInput` as we do in `MoneyRequestAmountInput`.

Reply:
I just realized that we have two refs coming out from `MoneyRequestAmountInput`, that shouldn't happen, let's remove the `moneyRequestAmountInputRef` prop and use the `forwardedRef` only and apply `useImperativeHandle` on it and add missing methods (e.g. focus, blur)